primitive-types/impls/serde/Cargo.toml

```diff
@@ -13,13 +13,13 @@ default = ["std"]
 std = ["serde/std"]
```

Comment:
missing update of `[package]` fields above

Reply:
done, and other crates under `primitive-types/impls` are missing as well, I also update them